.ci/containers/github-differ/compare_breaking_changes.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# input: two environment variables
+#  TPG_BREAKING - results of runing breaking change detector
+#   against tpg
+#  TPGB_BREAKING - results of runing breaking change detector
+#   against tpgb
+# output: echo to console
+#  message section cotaining: a header,
+#  tpg's unique messages, and all of tpgb's messages
+
+tpgUnique=""
+newline=$'\n'
+
+while IFS1= read -r tpgi; do
```

Comment:
I'm a little concerned about the complexity of this section - it uses a lot more bash logic than we generally make use of on our team. That being said, it should be fine temporarily, and we'll ideally replace it with a different language soon. But could you add a comment above this section explaining the logic & specifically calling out unusual syntax like `<<<`, `IFS`, and anything else that seems worth mentioning.